src/snowflake/snowpark/functions.py

```diff
@@ -11663,9 +11703,8 @@ def fl_get_content_type(e: ColumnOrName, _emit_ast: bool = True) -> Column:
         'text/csv'
     """
     function_name = "fl_get_content_type"
-    ast = build_function_expr(function_name, [e]) if _emit_ast else None
     col_input = _to_col_if_str(e, function_name)
-    return builtin(function_name, _ast=ast, _emit_ast=_emit_ast)(col_input)
+    return _call_function(function_name, col_input, _emit_ast=_emit_ast)
```

Comment:
why are we dropping ast here and in some other places down below?

Reply:
In all of these cases, the AST generation for the functions can be handled by `_call_function` directly as the inputs are simple (no conversions of constants to Snowpark `Literal`s), and the function name matches the SQL function name provided to `_call_function`.